.nvmrc

```diff
@@ -1 +1 @@
-12.22.6
+14.20.0
```

Comment:
Should we specify the whole version number here @zmaglica? It think just `14` works fine.

Someone using NVM without the specific version installed, running `nvm install 14` will always install the latest version. What do you think?

Reply:
Yeah, we can leave 14. The latest version will always be installed (14.20.0) anyway, so I will make a change to my PR.

Reply:
Thanks 👍